config.toml

```diff
@@ -107,10 +107,6 @@ copyright_linux =  "Copyright © 2020 The Linux Foundation ®."
 # First one is picked as the Twitter card image if not set on page.
 # images = ["images/project-illustration.png"]
 
-# Menu title if your navbar has a versions selector to access old versions of your site.
-# This menu appears only if you have at least one [params.versions] set.
-version_menu = "Versions"
```

Comment:
Does it hurt if we leave line 110-112 untouched?

Reply:
Yes, we can keep it untouched. Do I revert this deletion for the safe fallback or something?

Reply:
I'd suggest leave it as is.

Reply:
OK, I'll do so.